pkg/applier/manager.go

```diff
@@ -151,7 +150,7 @@ func (m *Manager) runWatchers(ctx context.Context) {
 				if dir.IsDirectory(event.Name) {
 					m.createStack(ctx, stacks, event.Name)
 				}
-			case fsnotify.Remove:
```

Comment:
Interesting. The unit test fails on Windows, whereas it works well on Linux. I suspect this is due to the implicit event order which is probably different between Windows and Linux. I can imagine how the Create event for the new name happening before the Rename event for the old name will screw things up. Maybe the fix is more involved than simply reacting on Rename events, as well.